docs/cluster-workflow-templates.md

```diff
@@ -140,7 +140,7 @@ argo submit https://raw.githubusercontent.com/argoproj/argo-workflows/master/exa
 The submit a `ClusterWorkflowTemplate` as a `Workflow`:
 
 ```bash
-argo submit --from clusterworkflowtemplate/workflow-template-submittable
+argo submit --from clusterworkflowtemplate/cluster-workflow-template-submittable
```

Comment:
good catch. confirmed this is the name: https://github.com/argoproj/argo-workflows/blob/4aa90111d20ac49d1b1150218bb92b2cd9dcee53/examples/cluster-workflow-template/clustertemplates.yaml#L136